doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,89 @@
+[id='proc_kogito-deploying-on-ocp-console']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the OpenShift web console
+
+After you create your {PRODUCT} services as part of a business application, you can use the {OPENSHIFT} web console to deploy your services. As an aid to deploy your services, you can also use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can check out the project, build it (including native compilation, if applicable), and deploy the resulting services.
+
+.Prerequisites
+* The application with your {PRODUCT} services is in a Git repository.
+* You have access to the OpenShift web console with `cluster-admin` permissions.
+
+.Procedure
+. In the OpenShift web console, go to *Operators* -> *OperatorHub* in the left menu, search for and select *Kogito Operator*, and follow the on-screen instructions to install the latest operator version.
++
+--
+On {OPENSHIFT} 3.11, the OperatorHub catalog is not available by default in the web console, so you must use the `oc` CLI to manually install the {PRODUCT} Operator for your defined project:
+
+.Manually installing {PRODUCT} Operator on OpenShift 3.11
```

Comment:
If we will keep this, I'd say to point to Strimzi (Kafka Operator) and Infinispan Operator install instructions as well. Kogito CLI won't install those on OCP 3.11.

Reply:
And you don't have the `Installed Operators` option on OCP 3.11 console. :disappointed: 
You have to see if it's a pod named `kogito-operator-hash` deployed.

Reply:
Actually, because we've decided to go with "OpenShift 4.2 or later", I've just removed this bit. Anyway simpler and focuses on the target OCP version for optimal user experience. My opinion. Okay with you?

Reply:
once we have the kubernetes integration fully working with operator we probably want to update this later.